src/widgets/communication/ui/communication.test.tsx

```diff
@@ -0,0 +1,63 @@
+import { cleanup, screen } from '@testing-library/react';
+import { Communication } from './communication';
+import { renderWithRouter } from '@/shared/__tests__/utils';
+import { DISCORD_LINKS, communicationText } from 'data';
+
+describe('Communication section', () => {
+  it('should render "Communication" section texts correctly', async () => {
+    const firstCourse = Object.keys(DISCORD_LINKS)[0] as keyof typeof DISCORD_LINKS;
+
+    const {
+      title,
+      subTitle,
+      firstParagraphFirstHalf,
+      discordLink,
+    } = communicationText.en;
+
+    renderWithRouter(<Communication courseName={firstCourse} />);
+    const titleElement = screen.getByText(title);
+    const subtitleElement = screen.getByText(subTitle);
+    const firstParagraphElement = screen.getByText(`${firstParagraphFirstHalf}`, { exact: false });
+    const linkElement = screen.getByText(discordLink);
+
+    expect(titleElement).toBeVisible();
+    expect(subtitleElement).toBeVisible();
+    expect(firstParagraphElement).toBeVisible();
+    expect(linkElement).toBeVisible();
+    expect(linkElement.getAttribute('href')).toMatch(DISCORD_LINKS[firstCourse]);
+  });
+
+  it('should render "Communication" section texts correctly on russian language', async () => {
+    const firstCourse = Object.keys(DISCORD_LINKS)[0] as keyof typeof DISCORD_LINKS;
+
+    const {
+      title,
+      subTitle,
+      firstParagraphFirstHalf,
+      discordLink,
+    } = communicationText.ru;
+
+    renderWithRouter(<Communication courseName={firstCourse} lang="ru" />);
+    const titleElement = screen.getByText(title);
+    const subtitleElement = screen.getByText(subTitle);
+    const firstParagraphElement = screen.getByText(`${firstParagraphFirstHalf}`, { exact: false });
+    const linkElement = screen.getByText(discordLink);
+
+    expect(titleElement).toBeVisible();
+    expect(subtitleElement).toBeVisible();
+    expect(firstParagraphElement).toBeVisible();
+    expect(linkElement).toBeVisible();
+    expect(linkElement.getAttribute('href')).toMatch(DISCORD_LINKS[firstCourse]);
+  });
```

Comment:
Can it be "combined" like this https://github.com/rolling-scopes/site/pull/540#discussion_r1773397202?

Reply:
Yeah, I understand what about you, but only for 2 test cases, this feels little to much, and is easier to just copy.
What do you think?

Reply:
We can leave it as is for now and discuss it during our next call

Reply:
fiex: https://github.com/rolling-scopes/site/pull/536/commits/0d7d349989efc00fe3c38d8008c139313c543afa